storagedriver/azure/azure.go

```diff
@@ -272,6 +270,20 @@ func (d *Driver) Delete(path string) error {
 	return nil
 }
 
+// URLFor returns a URL which may be used to retrieve the content stored at the given path.
```

Comment:
Hmm the documentation is linked at the used library's godoc: http://godoc.org/github.com/MSOpenTech/azure-sdk-for-go/clients/storage#BlobStorageClient.GetBlobSASURI

There are no extra config needed. It just returns a publicly accessible URL for the blob for a duration of `options["expiry"]` (or 20 min default). Does registry code or person reading the code need to know anything else?